src/aurora/ltrfile.h

```diff
@@ -0,0 +1,60 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  File for creating random names in the character generation.
+ */
+
+#ifndef AURORA_LTRFILE_H
+#define AURORA_LTRFILE_H
+
+#include "src/common/ustring.h"
+
+#include "src/aurora/aurorafile.h"
+
+namespace Aurora {
+
+/**
+ * LTR File, which is used when generating player names.
+ *
+ * informations about the file structure are taken from
+ * https://github.com/mtijanic/nwn-misc
+ */
+class LTRFile : public AuroraFile {
+public:
+	LTRFile(Common::UString);
+
+	Common::UString generateRandomName(unsigned int maxLetters);
+
+private:
+	struct LetterSet {
+		float start[28];
+		float mid[28];
+		float end[28];
+	};
+
+	LetterSet                           _singleLetters;
+	std::vector<LetterSet>              _doubleLetters;
```

Comment:
You need to `#include <vector>` then. `ustring.h` also does, so this compiles without, but still, include what you use.